docs/writing-rules/experiments/symbolic-propagation.md

```diff
@@ -27,3 +27,24 @@ If we tried to match the pattern `pandas.DataFrame(...).index.set_value(...)` ag
 Symbolic propagation is a generalization of [constant propagation](/writing-rules/data-flow/constant-propagation/) that addresses this limitation. It enables Semgrep to perform matching modulo variable assignments. Thus, Semgrep is then able to match both `test1` and `test2` with the same simple pattern. This feature needs to be enabled explicitly via rule `options:` by setting `symbolic_propagation: true`.
 
 <iframe src="https://semgrep.dev/embed/editor?snippet=JeBP" border="0" frameBorder="0" width="100%" height="432"></iframe>
+
+## Limitations
```

Comment:
Maybe specific header can help our search:

```suggestion
## Limitations of symbolic propagation
```

Reply:
@adamkvitek I hadn't done this because it looks a little strange in the page context and isn't done in other similar pages like [Join mode](https://semgrep.dev/docs/writing-rules/experiments/join-mode/overview/#limitations), but if you think it's worth it for search, I don't mind!

Reply:
OK! added that in my update, just wanted to check.